tests/test_lifelongclassificationforest.py

```diff
@@ -38,13 +38,13 @@ def test_correct_estimator(self):
         self.assertIs(l2f.n_estimators, rand)
         
     def test_correct_default_finite_sample_correction(self):
-        tmpDict = {"finite_sample_correction": False}
-        self.assertEqual(self.l2f.pl.default_voter_kwargs, tmpDict)
+        tmp_dict = {"finite_sample_correction": False}
+        self.assertEqual(self.l2f.pl.default_voter_kwargs, tmp_dict)
     
     def test_correct_finite_sample_correction(self):
         rand = random.randint(0, 1)
-        tmpDict = {"finite_sample_correction": bool(rand)}
```

Comment:
Is it necessary? I think the previous function serves the purpose. The naming is also not proper. If I am correct, I think you are not testing whether finite_sample_correction is done in the right way. You are checking a class attribute.

Reply:
Could you put your test in the prglearn/tests folder?

Reply:
I think test_correct_default_finite_sample_correction and test_correct_finite_sample_correction are doing the same thing. Am I right?

Reply:
Should I just test_correct_finite_sample_correction for the False case instead then?

Reply:
I'll add it to the proglearn/tests folder after

Reply:
Yeap, that makes sense.

Reply:
tested, pushed and moved with latest commit